app/js/actions/QuestionActions.js

```diff
@@ -22,6 +22,13 @@ QuestionActions = {
       data: data
     });
   },
+  recieveUserQuestions: data => {
+    waitForQuestionStore();
+    AppDispatcher.dispatch({
+      actionType: ZhishiConstants.RECEIVE_USER_QUESTIONS,
+      data: data
+    });
+  },
```

Comment:
Its a google style guide for one arrow function. It states that if the variable is just one that we should not use brackets.